### PR TITLE
Make the playground header fit in with the rest of the site

### LINKED ISSUE #3542: [OCD] Website and Playground headers are not the same height
image

(Also, Prettier doesn't seem to be the same font)

### PR DESCRIPTION
**Pull request by j-f1**
Fixes #3542.

screen shot 2017-12-24 at 11 49 56

[<sub><sub></sub></sub> Live preview](https://deploy-preview-3562--prettier.netlify.com/playground/)

### CODE REVIEW

**Comment by vjeux** [comment]
Since you're doing some great work, I've added you as collaborator to the repo. You can now merge PRs :) I'm sure you're going to be responsible with this power! Thanks a lot for your contributions!

**Comment by j-f1** [comment]
Thanks for adding me!

Do you have any specific rules about PR approvals, or is one approval by another collaborator sufficient?

**Comment by vjeux** [comment]
One approval by a collaborator is needed and sufficient to merge. In doubts, apply common sense :)

**Comment by azz** [comment]
Welcome @j-f1! :tada:

File: website/static/playground.js [hunk 1]
```diff
@@ -83,7 +83,25 @@ window.onload = function() {
...
+        link.href = "https://github.com/prettier/prettier/pull/" + match[1];
+        link.textContent = "PR #" + match[1];
+      } else {
+        if (prettierVersion.match(/\.0$/)) {
+          link.href =
+            "https://github.com/prettier/prettier/releases/tag/" +
+            prettierVersion;
+        } else {
+          link.href =
+            "https://github.com/j-f1/forked-prettier/blob/master/CHANGELOG.md#" +
```
Comment (azz) [inline website/static/playground.js]: Why link to your fork?

Comment (vjeux) [reply website/static/playground.js]: Oops, good catch!

**Pull request merged**